sonaranalyzer-dotnet/its/expected/Automapper/AutoMapper-025B9F93-7953-454B-AF29-E6A55B2AA58F-S4136.json

```diff
@@ -0,0 +1,421 @@
+{
+"issues":  [
+{
+"id":  "S4136",
+"message":  "All 'MapFrom' signatures should be adjacent",
+"location":  [
+{
+"uri":  "sources\Automapper\src\AutoMapper\Configuration\MemberConfigurationExpression.cs",
```

Comment:
this is actually a different method than the one at line 135, because it's generic. I think we should differentiate them